Ryujinx.HLE/HOS/Services/Sockets/Bsd/Impl/WinSockHelper.cs

```diff
@@ -90,6 +91,54 @@ static class WinSockHelper
             {0, 0}
         };
 
+        private static readonly Dictionary<int, LinuxError> _errorMapMacOs = new()
+        {
+            {35, LinuxError.EAGAIN},
```

Comment:
```suggestion
            { 35, LinuxError.EAGAIN },
```
It should be formatted like this for consistency, although `_errorMap` is also missing the spaces. Would be nice to fix both.